woocommerce/woocommerce-ios
# Login Reminder: add an action to log in with WP.com for site address error notiifcation

Part of #7318 


### Description


For Iteration 1 in pe5sF9-h9-p2, we want to have a secondary action "Login with WordPress.com" that redirects the user to WordPress.com login screen. This PR added this action with a tracking event.

### Testing instructions


Please update the 24 hours schedule to a few seconds or a minute in `AuthenticationManager` (search for "86400") to test this PR. You can test this PR in a simulator.

- Log out from the app if needed
- Tap "Enter Your Store Address"
- Enter an invalid store address or a store address that isn't a WP site --> an error screen should be shown
- Quickly put the app to the background since we don't support local notifications in the foreground for this iteration --> a local notification should be shown after the duration you set in `AuthenticationManager`
- Long press on the notification --> there should be a second action "Login with WordPress.com" in the menu
- Tap "Login with WordPress.com" --> it should open the app, navigates to the WP.com login screen, and an event should be logged `🔵 Tracked login_local_notification_tapped, properties: [AnyHashable("action"): "login_with_wpcom", AnyHashable("type"): "site_address_error"]`

### Screenshots


<img src="https://user-images.githubusercontent.com/1945542/181196052-ae8f0c21-3ca7-4d30-a228-64f4d687b4c2.png" width="300" />

https://user-images.githubusercontent.com/1945542/181196067-1651abf0-cca3-438d-b9cf-9d662c27ab80.mov



---
